.gitignore

```diff
@@ -108,5 +108,8 @@ lib/z.lib
 #########################
 lib/matplotlib/backends/web_backend/node_modules/
 lib/matplotlib/backends/web_backend/package-lock.json
-
+Scripts
+lib/.gitignore
+.gitignore
+pyvenv.cfg
```

Comment:
We definitely to not want to ignore the gitignore file!


I'm not sure where `Scripts` come from, but can you put the `pyvenv.cfg` in a new section (labeled "niche tooling files" or something like that)?